### ISSUE TITLE
«jaqbīaq»

### ISSUE THREAD

**Issue by uakci** [toaq/dictionary#61]
> ▯ is sufficiently the case for ▯ to be the case; ▯ is so much so that ▯ is the case.

This definition is worrying. Where did jaqbīaq₂ come from? And what is «jaq» supposed to suggest? (As far as I know, «jaqbīaq» ≈ «bıaq jảq», and it just so happens that (bıaq jảq)¹ ≡ bıaq¹ and (bıaq jảq)² ≡ bıaq², from which one can infer that jaqbīaq ≡ bıaq.)
I propose something like «shıubīaq» or «cabīaq».

**Comment by Ilmen-vodhr** [comment]
I personally never liked **jaqbīaq** and prefer [**cabīaq**](https://uakci.pl/toadua/##yEvkHzigx), which I have already entered in Toadua some time ago.

**Comment by solpahi** [comment]
> As far as I know, «jaqbīaq» ≈ «bıaq jảq», and it just so happens that (bıaq jảq)¹ ≡ bıaq¹ and (bıaq jảq)² ≡ bıaq², from which one can infer that jaqbīaq ≡ bıaq.

No, **biaq**<sub>2</sub> is defined as (c 1), while **jaqbīaq**<sub>2</sub> is (0 0).

**jaqbīaq** is "___ is sufficiently (much) the case for ___ to be the case", which is a combination of **jaq** and **bıaq**. 

I don't share the preference for **cabīaq**.

**Comment by uakci** [comment]
If so, it could be made into a kind of **huaq**:
> ▯ is sufficiently the case for ▯ to be the case; ▯ satisfies property ▯ so much so that it satisfies property ▯.

**Comment by Ilmen-vodhr** [comment]
> No, **biaq**2 is defined as (c 1), while **jaqbīaq**2 is (0 0).
> 
> **jaqbīaq** is "___ is sufficiently (much) the case for ___ to be the case", which is a combination of **jaq** and **bıaq**.

What I don't like about it is that the ``for ___ to be the case`` part pops out of nowhere, and the **jaq-** prefix doesn't seem to be related to that slot.

**Comment by Ilmen-vodhr** [comment]
I like @uakci's last proposal. However it doesn't allow the **bıaq/2** reading "▯ sufficiently satisfies property ▯``, unfortunately.
